bedrock/lightbeam/templates/lightbeam/lightbeam.html

```diff
@@ -107,15 +101,6 @@ <h2 class="page-slogan">{{_('How Lightbeam Works')}}</h2>
       </p>
     </figcaption>
   </figure>
-  <figure class="span4 contribute">
```

Comment:
Removing this from the page kinda leaves things off balance visually. Can I suggest you change the two remaining items to have a class of `span6` so they make better use of the available space?